### ISSUE TITLE
2011 General precinct results

### ISSUE THREAD

**Issue by dwillis** [openelections/openelections-data-ms#34]
Data entry/OCR from files [here](http://www.sos.ms.gov/Elections/electionResults/General.asp):

- [ ] Adams
- [ ] Alcorn
- [ ] Amite
- [ ] Attala
- [ ] Benton
- [ ] Bolivar
- [ ] Calhoun
- [ ] Carroll
- [ ] Chickasaw
- [ ] Choctaw
- [ ] Claiborne
- [ ] Clarke
- [ ] Clay
- [ ] Coahoma
- [ ] Copiah
- [ ] Covington
- [ ] Desoto
- [ ] Forrest
- [ ] Franklin
- [ ] George
- [ ] Greene
- [ ] Grenada
- [ ] Hancock
- [ ] Harrison
- [ ] Hinds
- [ ] Holmes
- [ ] Humphreys
- [ ] Issaquena
- [ ] Itawamba
- [ ] Jackson
- [ ] Jasper
- [ ] Jefferson Davis
- [ ] Jefferson
- [ ] Jones
- [ ] Kemper
- [ ] Lafayette
- [ ] Lamar
- [ ] Lauderdale
- [ ] Lawrence
- [ ] Leake
- [ ] Lee
- [ ] Leflore
- [ ] Lincoln
- [ ] Lowndes
- [ ] Madison
- [ ] Marion
- [ ] Marshall
- [ ] Monroe
- [ ] Montgomery
- [ ] Neshoba
- [ ] Newton
- [ ] Noxubee
- [ ] Oktibbeha
- [ ] Panola
- [ ] Pearl River
- [ ] Perry
- [ ] Pike
- [ ] Pontotoc
- [ ] Prentiss
- [ ] Quitman
- [ ] Rankin
- [ ] Scott
- [ ] Sharkey
- [ ] Simpson
- [ ] Smith
- [ ] Stone
- [ ] Sunflower
- [ ] Tallahatchie
- [ ] Tate
- [ ] Tippah
- [ ] Tishomingo
- [ ] Tunica
- [ ] Union
- [ ] Walthall
- [ ] Warren
- [ ] Washington
- [ ] Wayne
- [ ] Webster
- [ ] Wilkinson
- [ ] Winston
- [ ] Yalobusha
- [ ] Yazoo

**Comment by stedy** [comment]
Ok same plan for me as 2016 elections - start with Yazoo and work my way backwards

**Comment by dwillis** [comment]
@stedy thanks for your work on this! One thing I noticed; the files seem to include any of the state senate and state house races.

**Comment by stedy** [comment]
yeah just getting back into this after the holidays and I noticed your updated comment on this yesterday. Will add those offices from Panola going forward and then go back and update Pearl River to Yazoo

**Comment by dwillis** [comment]
Many thanks, and no rush.

**Comment by stedy** [comment]
I have been getting an HTTP 500 error for quite a while at the MS SOS site for 2011. Will keep trying

**Comment by dwillis** [comment]
Feel free to email the SOS.

**Comment by stedy** [comment]
Working on it, cannot find any contact info - right now almost every page returns a 404 :disappointed:

**Comment by dwillis** [comment]
Try Leah.Smith@sos.ms.gov

**Comment by stedy** [comment]
Emailed her and she said their IT would look into it, still waiting...

**Comment by dwillis** [comment]
@stedy looks like we're back in action!

**Comment by stedy** [comment]
Cool I am still getting an HTTP 500 but I will start checking it more often
